app/controllers/results_controller.rb

```diff
@@ -8,9 +8,21 @@ def index
       config.access_token_secret = ENV["TWITTER_ACCESS_SECRET"]
     end
 
-    if params[:search]
+    def find_center(tweets)
```

Comment:
Method definitions must not be nested. Use `lambda` instead.<br>Unused method argument - `tweets`. If it's necessary, use `_` or `_tweets` as an argument name to indicate that it won't be used. You can also write as `find_center(*)` if you want the method to accept any arguments but don't care about them.